behind-the-veil-siteroot/imports/ui/components/pages/home/HomePage.jsx

```diff
@@ -144,34 +144,27 @@ export const HomePage = () => {
             </div>
 
             {/* Container for the title text on the home page (positioned absolute) */}
-            <div className={"flex flex-row items-center justify-center w-full"}>
-                <div
-                    className={"absolute z-30 main-text text-white text-xl md:text-2xl lg:text-3xl xl:text-5xl " +
-                        "left-[16%] top-[45%] w-[68%] " +
-                        "sm:left-[16%] sm:top-[56.5%] sm:w-[68%] " +
-                        "md:left-[16%] md:top-[56%] md:w-[68%] " +
-                        "lg:left-[16%] lg:top-[52.5%] lg:w-[68%] " +
-                        "xl:left-[20%] xl:top-[54%] xl:w-[60%] "
-                    }>
-                    <p className="text-center"> Make Yourself Perfect</p>
-                    <p className="inline-flex items-center justify-center text-center gap-x-5 h-1/2 w-full">
-                        <SparklesIcon
-                            className="size-6 md:size-8 lg:size-10 xl:size-14 text-anything-yellow fill-anything-yellow"/>
-                        Behind The Veil
-                        <SparklesIcon
-                            className="size-6 md:size-8 lg:size-10 xl:size-14 text-anything-yellow fill-anything-yellow"/>
-                    </p>
-                </div>
-                {/*Background blur*/}
-                <div className="absolute rounded-full z-[19] blur-md
-                    border-main-blue border-2
-                    bg-gradient-to-r from-bg-gradient-end/85 to-bg-gradient-start/85
-                    left-[16%] top-[42.5%] w-[68%] h-[12%]
-                    sm:top-[54%] sm:h-[12%] sm:blur-md
-                    md:top-[52.5%] md:h-[15%] md:blur-lg
-                    lg:top-[46.5%] lg:h-[22%] lg:blur-xl
-                    xl:top-[47.5%] xl:h-[25%] xl:blur-2xl">
-                </div>
+            <div
```

Comment:
oh btw, just wondering are we going with the fuzzier design in figma? this is what this branch looks like
![image](https://github.com/Monash-FIT3170/Behind-The-Veil/assets/95511987/8901d0f4-31ab-4770-baa2-67269a1f428c)